builder-support/dockerfiles/Dockerfile.debbuild-prepare

```diff
@@ -40,7 +42,7 @@ RUN cp /sdist/pdns-recursor-${BUILDER_VERSION}.tar.bz2 pdns-recursor_${BUILDER_V
 @ENDIF
 
 @IF [ -n "$M_dnsdist$M_all" ]
-RUN tar xvf /sdist/dnsdist-${BUILDER_VERSION}.tar.bz2
+RUN tar xvf /sdist/dnsdist-${BUILDER_VERSION}.tar.xz
 # create copy of source tarball with name that dpkg-source requires
-RUN cp /sdist/dnsdist-${BUILDER_VERSION}.tar.bz2 dnsdist_${BUILDER_VERSION}.orig.tar.bz2
+RUN cp /sdist/dnsdist-${BUILDER_VERSION}.tar.xz dnsdist_${BUILDER_VERSION}.orig.tar.zx
```

Comment:
`xz` and `zx` ? but also see general comment wrt tarball name

Reply:
Oops, that's indeed a typo. I'll fix for now, pending a decision on the actual tarball format.